packages/expo-screen-orientation/src/ScreenOrientation.types.ts

```diff
@@ -47,12 +47,6 @@ export enum WebOrientation {
   LANDSCAPE_SECONDARY = 'landscape-secondary',
```

Comment:
Discussed with @ide that we would like to change all these string enums to integers - that would probably also require some mappings on the native side.

Reply:
Before changing the strings to integers, I would suggest removing LANDSCAPE and PORTRAIT from Orientation enum. I think it looks misleading because the device cannot be in those orientations (only in left-right or up-down). It was needed as long as the iOS traitCollection was used, so after some refactoring, maybe it would be possible to get rid of them but I need makers' view on this.

Reply:
Another thing is proper device rotation in lockPlatformAsync, maybe using the mask to decide which orientation to choose in enforceDesiredDeviceOrientationWithOrientationMask, because for now, device can be in forbidden orientation after locking with lockPlatformAsync, it changes only after manually rotating to supported orientation.